### DISCUSSION TITLE
Not Able To Add Contolnet

### DISCUSSION THREAD

**Discussion by RFedz007989** [community/community#123411] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi,

I am not able to use Controlnet. When I click on Controlnet icon it's not highlighted, as in previous old versions there were command controls to add or de-select Controlnet, but in latest Ver 4.2.0 there is no such thing available. can you Pls Assist me on this.

Thank You

**Comment by ebndev** [comment]
Hi @RFedz007989,

Thanks for being a part of the GitHub Community, we're glad you're here!

It looks like you're looking for support from Contolnet. You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page.

While it's possible other users may have encountered this issue, we want to make sure you’re getting the best support you can and that your message goes where you intended- but this may not be the right place.

Best of luck!

**Reply by RFedz007989** [reply]
Thanks a lot. can you Pls tell where should I discuss this issue, as I am not aware where I will get the exact solution for this issue.
Thank You
